cpp/tests/prims/mg_count_if_e.cu

```diff
@@ -0,0 +1,373 @@
+
+    raft::comms::initialize_mpi_comms(&handle, MPI_COMM_WORLD);
+    auto& comm           = handle.get_comms();
+    auto const comm_size = comm.get_size();
+    auto const comm_rank = comm.get_rank();
+
+    auto row_comm_size = static_cast<int>(sqrt(static_cast<double>(comm_size)));
+    while (comm_size % row_comm_size != 0) {
+      --row_comm_size;
+    }
+    cugraph::partition_2d::subcomm_factory_t<cugraph::partition_2d::key_naming_t, vertex_t>
+      subcomm_factory(handle, row_comm_size);
+
+    // 2. create MG graph
+
+    if (cugraph::test::g_perf) {
+      CUDA_TRY(cudaDeviceSynchronize());  // for consistent performance measurement
+      handle.get_comms().barrier();
+      hr_clock.start();
+    }
+    auto [mg_graph, d_mg_renumber_map_labels] =
+      input_usecase.template construct_graph<vertex_t, edge_t, weight_t, store_transposed, true>(
+        handle, prims_usecase.test_weighted, true);
+
+    if (cugraph::test::g_perf) {
+      CUDA_TRY(cudaDeviceSynchronize());  // for consistent performance measurement
+      handle.get_comms().barrier();
+      double elapsed_time{0.0};
+      hr_clock.stop(&elapsed_time);
+      std::cout << "MG construct_graph took " << elapsed_time * 1e-6 << " s.\n";
+    }
+
+    auto mg_graph_view = mg_graph.view();
+
+    // 3. run MG count_if_e
+
+    const int hash_bin_count = 5;
+    const int initial_value  = 0;
+
+    auto property_initial_value = generate<result_t>::initial_value(initial_value);
+    using property_t            = decltype(property_initial_value);
+    auto vertex_property_data =
+      generate<result_t>::vertex_property((*d_mg_renumber_map_labels), hash_bin_count, handle);
+    auto col_prop =
+      generate<result_t>::column_property(handle, mg_graph_view, vertex_property_data);
+    auto row_prop = generate<result_t>::row_property(handle, mg_graph_view, vertex_property_data);
+
+    if (cugraph::test::g_perf) {
+      CUDA_TRY(cudaDeviceSynchronize());  // for consistent performance measurement
+      handle.get_comms().barrier();
+      hr_clock.start();
+    }
+
+    auto result = count_if_e(
+      handle,
+      mg_graph_view,
+      row_prop.device_view(),
+      col_prop.device_view(),
+      [] __device__(auto row, auto col, weight_t wt, auto row_property, auto col_property) {
+        return row_property < col_property;
+      });
+    if (cugraph::test::g_perf) {
+      CUDA_TRY(cudaDeviceSynchronize());  // for consistent performance measurement
+      handle.get_comms().barrier();
+      double elapsed_time{0.0};
+      hr_clock.stop(&elapsed_time);
+      std::cout << "MG count if e took " << elapsed_time * 1e-6 << " s.\n";
+    }
+
+    //// 4. compare SG & MG results
+
+    if (prims_usecase.check_correctness) {
+      cugraph::graph_t<vertex_t, edge_t, weight_t, store_transposed, false> sg_graph(handle);
+      std::tie(sg_graph, std::ignore) =
+        input_usecase.template construct_graph<vertex_t, edge_t, weight_t, store_transposed, false>(
```

Comment:
I just changed this in a PR that got merged.  If you merge the latest 21.10 you'll find this doesn't compile.

You need to change this call to something like:
`cugraph::test::construct_graph<vertex_t, edge_t, weight_t, store_transposed, false>(handle, input_usecase, true, false)`